models/articles-models.js

```diff
@@ -55,3 +62,36 @@ exports.fetchCommentsByArticleId = (article_id) => {
     return result[1].rows;
   });
 };
+
+exports.insertNewComment = (article_id, newComment) => {
+  const validColumns = ["username", "body"];
+
+  for (const key of validColumns) {
+    if (!newComment.hasOwnProperty(key)) {
+      return Promise.reject({ msg: "Missing required username or body" });
+    }
+  }
```

Comment:
Instead of restricting comments to only objects which have a username or body, we should check that the username or body exist, this allows our endpoint to be more extensibile in the future if we need it.